src/main/java/com/melon/portfoliomanager/exceptions/NullFieldException.java

```diff
@@ -0,0 +1,8 @@
+package com.melon.portfoliomanager.exceptions;
+
+public class NullFieldException extends RuntimeException{
```

Comment:
I think this exception is not used anymore so we can safely delete the whole file.